README.md

```diff
@@ -4,11 +4,12 @@ This package provides a set of algorithms for data clustering.
 
 [![0.6](http://pkg.julialang.org/badges/Clustering_0.6.svg)](http://pkg.julialang.org/?pkg=Clustering&ver=0.6)
 [![0.7](http://pkg.julialang.org/badges/Clustering_0.7.svg)](http://pkg.julialang.org/?pkg=Clustering&ver=0.7)
-[![1.0](http://pkg.julialang.org/badges/Clustering_1.0.svg)](http://pkg.julialang.org/?pkg=Clustering&ver=1.0)
```

Comment:
FWIW 0.7 badges have been broken for ages and as of recently (yet unannounced) there are no badge updates at all right now.

Reply:
It's a pity. I hope there would be some alternative soon. The badges were conveying some impression of a coherent ecosystem. I'll remove them for now.